README.md

```diff
@@ -5,11 +5,94 @@
 [![Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/projectcaluma/emeis)
 [![License: GPL-3.0-or-later](https://img.shields.io/github/license/projectcaluma/emeis)](https://spdx.org/licenses/GPL-3.0-or-later.html)
 
-user management
+A generic user management service.
+
+The word emeis ([e̞ˈmis]) comes from greek (εμείς) and means `we`.
 
 [Original RFC that led to emeis](docs/original_emeis_rfc.md)
 
-## Getting started
+### Configuration
+
+It's important to configure the app before the first start. When running for the first
+time, there is a data migration, that depends on settings from the config.
+
+emeis is a [12factor app](https://12factor.net/) which means that configuration is
+stored in environment variables.
+Different environment variable types are explained at
+[django-environ](https://github.com/joke2k/django-environ#supported-types).
+
+#### Common
+
+A list of configuration options which you might need to configure to get emeis started in your environment.
+
+##### General
+* `SECRET_KEY`: A secret key used for cryptography. This needs to be a random string of a certain length. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-SECRET_KEY).
+* `ALLOWED_HOSTS`: A list of hosts/domains your service will be served from. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts).
+
+##### Database
+* `DATABASE_ENGINE`: Database backend to use. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-DATABASE-ENGINE). (default: django.db.backends.postgresql)
+* `DATABASE_HOST`: Host to use when connecting to database (default: localhost)
+* `DATABASE_PORT`: Port to use when connecting to database (default: 5432)
+* `DATABASE_NAME`: Name of database to use (default: emeis)
+* `DATABASE_USER`: Username to use when connecting to the database (default: emeis)
+* `DATABASE_PASSWORD`: Password to use when connecting to database
+
+##### i18n
+* `LANGUAGE_CODE`: Language code of the default language (default: en)
+* `LANGUAGES`: A list of supported languages (default: \[en\])
+
+##### Bootstrapping
+* `ADMIN_USERNAME`: Username of the admin user (will be created on first run - default: admin)
+* `ADMIN_ROLE_SLUG`: Slug of the admin role (will be created on first run - default: admin)
+* `ADMIN_SCOPE_NAME`: Name of the admin scope (will be created on first run - default: admin)
+
+#### Additional options
+
+Additional options you might want to configure
+
+##### OIDC
+
+For OIDC, a customized [mozilla-django-oidc](https://github.com/mozilla/mozilla-django-oidc) is used.
+
+Most of the settings below are documented in it's [respective documentation](https://mozilla-django-oidc.readthedocs.io/en/stable/settings.html).
+
+* `OIDC_OP_USER_ENDPOINT`: Url of userinfo endpoint as [described](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
```

Comment:
```suggestion
* `OIDC_OP_USER_ENDPOINT`: Url of userinfo endpoint (see [spec](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo))
```

Reply:
Done. This should probably also be changed in Caluma.